lib/render/user.php

```diff
@@ -181,54 +181,54 @@ function RenderSiteAwards($userAwards)
             } elseif ($awardType == 2) { //    Developed a number of earned achievements
                 $tooltip = "Awarded for being a hard-working developer and producing achievements that have been earned over " . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . " times!";
 
-                $imagepath = getenv('ASSET_URL') . "/Images/_Trophy" . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . ".png";
+                $imagepath = "/Images/_Trophy" . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . ".png";
```

Comment:
Is there a reason why we're foregoing the use of the `ASSET_URL` environment variable?

Reply:
These are the only site award images that are still being served from Amazon S3. In Discord @luchaos agreed to just have these served from the RA server as well. Not sure if the plan is to serve all the remaining images from the RA server as well, or possibly all but the achievement badges.

Reply:
Ahh, so the `ASSET_URL` variable holds an external URL? I guess I missed that part.

Personally, I would probably just update the value to not reference Amazon instead of removing all instances of it. But I'm not sure if there are strong feelings one way or another regarding the benefits of keeping it. I'll let other devs weigh in on this one.

Reply:
we still want to host achievements on s3 for now - a test whether the increased server load by serving all images on ra.org has yet to be performed.
not ideal but removing the ASSET_URL is the way to go for now 👍